frontend/main.js

```diff
@@ -49,6 +49,16 @@ console.info('CONTRACTS_VERSION:', process.env.CONTRACTS_VERSION)
 console.info('LIGHTWEIGHT_CLIENT:', process.env.LIGHTWEIGHT_CLIENT)
 console.info('NODE_ENV:', process.env.NODE_ENV)
 
+if (process.env.CI) {
```

Comment:
Add logging information on CI to get a better stack trace and identify missing error handling.